### PR TITLE
[PyTorch] quantize quantized::linear bias with input value

### PR DESCRIPTION
**Pull request by jackm321**
Summary:
Choose bias quantization params based on value in the bias. There are still mismatched between PyTorch and Glow quantization but they are much smaller now (see test plan).

Documentation:
n/a

Test Plan:
enable `test_quantized_linear_random_input()` `bias = torch.tensor([1, 2, 3, 4, 5, 6, 7], dtype=torch.float) * 0.1` and measure the diff between PyTorch and Glow results.

before
```
 tensor([[-0.0769, -0.1538, -0.3077, -0.4615, -0.5385,  0.3846,  0.2308],
        [-0.1538, -0.1538, -0.3077, -0.3846, -0.4615,  0.3846,  0.3077],
        [-0.0769, -0.1538, -0.3077, -0.3846, -0.4615,  0.3846,  0.3077],
        [-0.0769, -0.2308, -0.3077, -0.3846, -0.5385,  0.3846,  0.3077],
        [-0.1538, -0.1538, -0.3077, -0.3846, -0.5385,  0.4615,  0.3077],
        [-0.0769, -0.2308, -0.3077, -0.4615, -0.4615,  0.3846,  0.3077],
        [-0.1538, -0.1538, -0.3077, -0.3846, -0.4615,  0.3846,  0.3077]])
```
after
```
 tensor([[-0.0769,  0.0769,  0.0000, -0.0769,  0.0000,  0.0769,  0.0000],
        [ 0.0000,  0.0769,  0.0769,  0.0000,  0.0769,  0.0000,  0.0000],
        [-0.0769,  0.0769,  0.0000,  0.0000, -0.0769,  0.0000,  0.0000],
        [ 0.0000,  0.0000,  0.0000, -0.0769,  0.0000,  0.0000,  0.0000],
        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000],
        [-0.0769,  0.0000,  0.0000,  0.0000,  0.0769,  0.0000,  0.0000],
        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.0769,  0.0769,  0.0000]])
```

### CODE REVIEW

**Comment by jackm321** [comment]
For the remaining errors. The first thing I'm going to try next is creating an FC instruction in Glow and using that instead of Matmul + BatchedAdd for quantized FC because right now there is a precision loss when we convert matmul output to int8 then apply the bias instead of doing the bias addition in int32 then going to int8

**Comment by jackm321** [comment]
@yinghai not yet, but one we should look into is adding a quantized fc op to the Glow interpreter so that the int32 bias is added to the accumulated sum before requantizing back to int8.

**Comment by yinghai** [comment]
Sounds good. Thanks!

**Comment by jackm321** [comment]
@zrphercule well from the above differences, we can see that the errors are much lower now but not zero so I think we need to keep working on this
